src/VisualStudio/Core/Def/Implementation/Progression/GraphBuilder.cs

```diff
@@ -388,8 +388,8 @@ private static void UpdateLabelsForNode(ISymbol symbol, Solution solution, Graph
                 case SymbolKind.Property:
                     node.Label = symbol.MetadataName;
 
-                    var propertySymbol = symbol as IPropertySymbol;
-                    if (propertySymbol != null && propertySymbol.IsIndexer && LanguageNames.CSharp == propertySymbol.Language)
+                    var propertySymbol = (IPropertySymbol)symbol;
+                    if (propertySymbol.IsIndexer && LanguageNames.CSharp == propertySymbol.Language)
```

Comment:
:memo: This one wasn't pointed out, but I believe is valid and now matches the pattern for the rest of this switch statement.